src/components/CreateToken.tsx

```diff
@@ -69,8 +69,17 @@ export const CreateToken: FC = () => {
       const mintSignature = await sendTransaction(createMintTransaction, connection, {signers: [mintKeypair]});
       await connection.confirmTransaction(mintSignature);
       await sendTransaction(createMetadataTransaction, connection);
+      console.log(mintKeypair.publicKey.toBase58())
   }, [publicKey, connection, sendTransaction]);
 
+  // const getMetaData = useCallback(async () => {
```

Comment:
Try not to leave comments in a PR. Can make the code messy long term

Reply:
Whoops, I forgot I had these commented out sections. It was from some testing I was doing. I’ll get them removed.